superset-frontend/cypress-base/cypress/e2e/dashboard/utils.ts

```diff
@@ -24,9 +24,9 @@ export const WORLD_HEALTH_CHARTS = [
   { name: '% Rural', viz: 'world_map' },
   { name: 'Most Populated Countries', viz: 'table' },
   { name: "World's Population", viz: 'big_number' },
-  { name: 'Growth Rate', viz: 'line' },
+  { name: 'Growth Rate', viz: 'echarts_timeseries_line' },
```

Comment:
Not a problem, but maybe we can rename the new ones at some point since they support categorical and are no longer timeseries charts, really.

Reply:
Yeah. We should definitely do that in the future.